readme.md

```diff
@@ -170,19 +170,19 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** You may notice that you can add custom error messages to individual fields failing. Try adding these and take a look at the error message received. How can you make use of those specific messages for each field?
 
-* **Your Answer:**
+* **Your Answer:** more specific error handling
```

Comment:
Right, you can add more specific messages to each validation. You can access these custom messages by using `try/catch` or `.catch()` and then digging into the provided error.